tests/aws/services/lambda_/test_lambda.py

```diff
@@ -949,12 +966,29 @@ def test_lambda_url_echo_invoke(self, create_lambda_function, snapshot, aws_clie
         # TODO: add more cases
         result = safe_requests.post(url, data="text", headers={"Content-Type": "text/plain"})
         assert result.status_code == 200
-        event = json.loads(result.content)["event"]
-        assert event["body"] == "text"
-        assert event["isBase64Encoded"] is False
 
-        result = safe_requests.post(url)
-        event = json.loads(result.content)["event"]
+        if invoke_mode != "RESPONSE_STREAM":
+            event = json.loads(result.content)["event"]
+            assert event["body"] == "text"
+            assert event["isBase64Encoded"] is False
+
+            result = safe_requests.post(url)
+            event = json.loads(result.content)["event"]
+
+        else:
+            response_chunks = []
+            for chunk in result.iter_content(chunk_size=1024):
+                if chunk:  # Filter out keep-alive new chunks
+                    response_chunks.append(chunk.decode("utf-8"))
+
+            # Join the chunks to form the complete response string
+            complete_response = "".join(response_chunks)
+
+            response_json = json.loads(complete_response)
+            event = json.loads(response_json["body"])["event"]
+            # TODO the chunk-event actually contains a key "body": "text" - not sure if we need more/other validation here
```

Comment:
yeah, we should probably do better snapshot and behavior testing once we actually implement streaming in LocalStack.